src/components/ItemCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { ReactComponent as HeartIcon } from "../assets/favorite.svg";
+import noImage from "../assets/no-image.png";
+import styles from "./ItemCard.module.css";
+
+function ItemCard({ className, item }) {
+  const thumbnailUrl = "https://example.com/...";
```

Comment:
상수라서 컴포넌트 밖에 선언해주시는 걸 추천드려요!
```suggestion
  const THUMBNAIL_URL = "https://example.com/..." as const;
```